query.go

```diff
@@ -23,6 +23,10 @@ func parseOTRQueryMessage(msg ValidMessage) []int {
 
 		if len(versions) > 0 && versions[0] == 'v' {
 			for _, c := range versions {
+				r := rune(c)
+				if r == '?' || r == ' ' || r == '\n' || r == '\r' || r == '\t' {
```

Comment:
This does too many unnecessary checks. The only thing that should be checked is when the occurrence of '?' happens.